elixirkit/lib/elixirkit.ex

```diff
@@ -0,0 +1,5 @@
+defmodule ElixirKit do
+  def start do
+    {:ok, _} = ElixirKit.Server.start_link(self())
```

Comment:
Why do we need this function? 🤔

Reply:
Did you mean LivebookApp should call Server.start_link directly?

Reply:
I am trying to understand why call this is not part of the ElixirKit supervision tree. Is this a PubSub? If so, would it be better to use the Registry? Or can we start several of these and, if so, shouldn't we have them in a DynamicSupervisor?

Reply:
Thanks, I changed this to:

```elixir
Supervisor.start_child(ElixirKit.Supervisor, {ElixirKit.Server, self()})
```